api/graphql/schema.graphql

```diff
@@ -108,7 +108,7 @@ type User {
     # Profile Status
     isProfileComplete: Boolean
     expectedGenericJobTitles: [GenericJobTitle] @belongsToMany
-
+    priorityWeight: Int @rename(attribute: "priority_status")
```

Comment:
Hehe, let's use priority weight or priority status (I'm fine with either) but let's use just one. :-)

Reply:
makes sense to me!
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3811/commits/992c3f93043b82be2bfe6bc740812da2a3e0bc1b